test/spec/geo/geometry-views/shared-tests-for-path-views.js

```diff
@@ -98,6 +98,31 @@ module.exports = function (Path, MapView, PathView) {
       });
     });
 
+    describe('when the point triggers a dblclick event', function () {
+      beforeEach(function () {
+        var pointToDelete = new Point({
+          latlng: [
+            -40,
+            40
+          ]
+        });
+
+        this.geometry.setCoordinates([
+          [ -10, 10 ], [ 10, 20 ], [ 30, 40 ]
+        ]);
+
+        this.geometry.addPoint(pointToDelete);
+
```

Comment:
Just a minor thing:

I would add a pre-spec to clarify the test

```javascript
expect(this.geometry.getCoordinates()).toEqual([[ -10, 10 ], [ 10, 20 ], [ 30, 40 ], [-40, 40]); // <---
pointToDelete.trigger('dblclick');
expect(this.geometry.getCoordinates()).toEqual([[ -10, 10 ], [ 10, 20 ], [ 30, 40 ]);
```

Reply:
🙇

Reply:
👍